src/zimwriterfs/zimcreatorfs.cpp

```diff
@@ -159,20 +159,41 @@ void ZimCreatorFS::visitDirectory(const std::string& path)
   closedir(directory);
 }
 
-void ZimCreatorFS::addArticle(const std::string& path)
+void ZimCreatorFS::addFile(const std::string& path)
 {
-  auto farticle = std::make_shared<FileArticle>(this, path);
```

Comment:
farticle - I feel sad that such a funny identifier is removed from the source code